WBooks/Screens/BookDetail/BookDetailFullViewController.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  BookDetailFullViewController.swift
+//  WBooks
+//
+//  Created by Federico Agustin Diaz on 06/08/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import UIKit
+import WolmoCore
+
+class BookDetailFullViewController: UIViewController {
+    
+    private let _view: BookDetailFullView = BookDetailFullView.loadFromNib()!
+    private let _detailHeaderView: BookDetailView = BookDetailView.loadFromNib()!
+    
+    var bookDetailViewModel: BookDetailFullViewModel!
+    
+    convenience init(with bookModel: BookDetailFullViewModel) {
+        self.init()
+        bookDetailViewModel = bookModel
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        title = "BOOK_DETAIL".localized()
+        
+        configureTableView()
+        
+        initBookDetailTableViewModel()
+        
+        let backButton = UIBarButtonItem.backButton(for: self, action: #selector(backButtonPressed))
+        navigationItem.leftBarButtonItems = [backButton]
+    }
+    
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        
+        _detailHeaderView.setup(with: bookDetailViewModel.bookModel)
+    }
+    
+    @objc func backButtonPressed() {
+        navigationController?.popViewController(animated: true)
+    }
+    
+    override func loadView() {
+        _detailHeaderView.delegate = self
+        _view.detailHeaderView.addSubview(_detailHeaderView)
+        view = _view
+    }
+    
+    // MARK: - Private
+    private func initBookDetailTableViewModel() {
+        
+        bookDetailViewModel.reloadViewClosure = { [weak self] () in
+            DispatchQueue.main.async {
+                self?._view.detailTable.reloadData()
+            }
+        }
+        
+        bookDetailViewModel.loadComments(for: bookDetailViewModel.bookModel, onErrorComments: { [weak self] (error) in
+            DispatchQueue.main.async {
+                self?.showAlertMessage(message: error.localizedDescription)
+            }
+        })
+    }
+    
+    private func configureTableView() {
+        _view.detailTable.delegate = self
+        _view.detailTable.dataSource = self
+        _view.configureDetailTableView()
+        let commentBookNib = UINib.init(nibName: "BookCommentCellView", bundle: nil)
+        _view.detailTable.register(commentBookNib, forCellReuseIdentifier: "BookCommentCellView")
```

Comment:
Can you use wolmoCore for registering the cell?

Reply:
It gives me an error, which doesnt happen with BookCell
Cannot invoke 'register' with an argument list of type '(cell: BookCommentCellView.Type)'

Reply:
`BookCommentCellView`  needs to be `Nibloadable`